harness/determined/cli/trial.py

```diff
@@ -14,8 +13,8 @@
 from determined.cli.master import format_log_entry
 from determined.common import api
 from determined.common.api import authentication, bindings
-from determined.common.declarative_argparse import Arg, ArgsDescription, Cmd, Group
-from determined.experimental import Determined
+from determined.common.declarative_argparse import Arg, ArgsDescription, Cmd, Group, string_to_bool
+from determined.experimental.client import CheckpointSortBy, Determined, OrderBy
```

Comment:
please stick to the google style guide when adding new imports.

Reply:
ha! I like it.

Ok, brb..

Reply:
It's more than an hour or two to migrate the whole CLI and I want to get this fix out today, so I just made the minimum change.

Reply:
sure, also arguably this fix and that migration shouldn't be in the same pr anyway